.github/workflows/scheduled_mdlinkcheck.yml

```diff
@@ -0,0 +1,34 @@
+name: Check Markdown links
+
+on: 
+  schedule:
+  # Run on the 1st and 15th of every month at 2:00 AM 
+  - cron: "0 2 1,15 * *"
```

Comment:
can we run this weekly? maybe every Mon morning at 2:00 am?

Reply:
I updated it to run weekly.